src/armTemplates/resources/functionApp.ts

```diff
@@ -129,8 +147,8 @@ export class FunctionAppResource implements ArmResourceTemplateGenerator {
     const resourceConfig: FunctionAppConfig = {
       ...config.provider.functionApp,
     };
-    const { functionRuntime } = config.provider;
-
+    const { functionRuntime, os } = config.provider;
+    const linux = os === FunctionAppOS.LINUX;
```

Comment:
can we add some parenthesis?

Reply:
Parentheses would seem redundant to me, as there is only one operation happening here. Just capturing the boolean in a variable.

Reply:
I was actually confused about what this does until you explain it’s about capturing Boolean in a variable, so I don’t think it’s that redundant

Reply:
That makes sense, I can be more expressive in the naming. I changed it to `isLinuxRuntime` to be more explicit.

I still don't think parentheses would be appropriate, since they're more for defining order of operations, and this is only one operation in the statement. Just like you wouldn't do `const a = (b + c);`